nautobot/core/templates/admin/base.html

```diff
@@ -27,10 +27,10 @@
           href="{% static 'flatpickr-4.6.9/themes/light.min.css' %}"
           onerror="window.location='{% url 'media_failure' %}?filename=flatpickr-4.6.9/themes/light.min.css'">
     <link rel="stylesheet" id="dark-theme"
-          href="{% static 'css/dark.css' %}?v{{ settings.VERSION }}"
+          href="{% versioned_static 'css/dark.css' %}"
           onerror="window.location='{% url 'media_failure' %}?filename=css/dark.css'"{% if request.COOKIES|get_item:"theme" != 'dark' %} disabled="disabled"{% endif %} />
     <link rel="stylesheet" id="base-theme"
-          href="{% static 'css/base.css' %}?v{{ settings.VERSION }}"
+          href="{% versioned_static 'css/base.css' %}"
           onerror="window.location='{% url 'media_failure' %}?filename=css/base.css'">
     <link rel="apple-touch-icon" sizes="180x180" href="{% static 'img/nautobot_icon_180x180.png' %}">
```

Comment:
Not directly related to this PR but it's odd that the admin template isn't using `custom_branding_or_static` for the icons here and below. Wouldn't be a bad idea to fix that.

Reply:
Related to that, we *probably* should use `versioned_static` in the `custom_branding_or_static` function for non-custom static files, just in case we change the icons/logos at some point in the future.